src/components/Modal.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  
+  <link rel="stylesheet" href="../styles/root.css">
+</head>
+<body>
+
+<!---------------------------------Modal Component --------------------------------->
+
+<div class="ModalMainContainer padding-MainContainer">
+
+  <div class="TypographyContainerOne">
+    <h1 class="h1Tag">Modal</h1>
+  </div>
+
+  <div class="TypographyContainerTwo paddingOtherContainer">
+    <h3 class="h3Tag">Modal with close button</h3>
+    <p class="descriptionOne discriptionText ">An Modal displays a short, important message in a way that attracts the user's attention without interrupting the user's task.</p>
+
+    <div class="DOMContainer flex-row-center">
+
+      <div class="TypographyDomMainContainer">
+        <div class="TypographyDomContainer TypoHtag TypographyError flex-column-start box-shadow">
+         
+          <button id="myBtn">Open Modal</button>
```

Comment:
Give an example of what the modal looks like. Then add a button for the demo.
Also, this open modal button is not working.

Reply:
Okay I'll work on that thank you.